include/tinyfsm.hpp

```diff
@@ -32,14 +32,25 @@
  *
  * Author:
  * Axel Burri <axel@tty0.ch>
+ * incloon
  * ---------------------------------------------------------------------
  */
 
 #ifndef TINYFSM_HPP_INCLUDED
 #define TINYFSM_HPP_INCLUDED
 
-#ifndef TINYFSM_NOSTDLIB
```

Comment:
Removing the possibility to not depend on type_traits means that this library no longer can be used on Arduino.